kmip/pie/api.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2015 The Johns Hopkins University/Applied Physics Laboratory
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import abc
+import six
+
+
+@six.add_metaclass(abc.ABCMeta)
+class KmipClient:
```

Comment:
We won't be changing the name of `KMIPProxy`. That's static for now. We're trying to finalize the name of the new `KmipClient` (see https://github.com/OpenKMIP/PyKMIP/pull/54). It provides a simpler interface and wraps the original `KMIPProxy`.